src/applications/claims-status/components/StemDeniedDetails.jsx

```diff
@@ -10,100 +15,151 @@ const StemDeniedDetails = ({
+                  <ol>
+                    <li>
+                      Be enrolled in an eligible STEM undergraduate degree,{' '}
+                      <strong>or</strong>
+                    </li>
+                    <li>
+                      Have an eligible STEM undergraduate degree and be pursuing
+                      a teaching degree
+                    </li>
+                  </ol>
+                </li>
+              </ul>
             </li>
-          </ul>
-        </li>
+          )}
       </ul>
       <h3 className="vads-u-font-family--sans vads-u-margin-bottom--0">
         You met these criteria for the Rogers STEM Scholarship:
       </h3>
       <ul className="stem-ad-list">
-        <li className="stem-ad-list-item">
-          You are eligible for Post 9/11 GI Bill benefits
-          <ul className="stem-ad-list-secondary">
-            <li className="stem-ad-list-item">
-              By law, all Rogers STEM scholarship recipients must be eligible
-              for Post 9/11 GI Bill (Chapter 33) benefits. (38 U.S. Code §
-              3320).
-            </li>
-            <li className="stem-ad-list-item">
-              We reviewed your service and school data records currently on
-              file. As of [{date}
-              ], you are eligible for Post 9/11 GI Bill benefits.
-            </li>
-          </ul>
-        </li>
+        {remainingEntitlement && (
+          <li className="stem-ad-list-item">
+            You're eligible for Post-9/11 GI Bill benefits
+            <ul className="stem-ad-list-secondary">
+              <li className="stem-ad-list-item">
+                According to your service and school data records on file, as of{' '}
+                {date}, you're eligible for Post-9/11 GI Bill benefits.
+              </li>
+              <li className="stem-ad-list-item">
+                By law, all Rogers STEM scholarship recipients must be eligible
+                for Post-9/11 GI Bill (Chapter 33) benefits. (38 U.S. Code §
+                3320).
+              </li>
+            </ul>
+          </li>
+        )}
+        {remainingEntitlement <= 180 && (
+          <li className="stem-ad-list-item">
+            You meet the benefit requirements for the Rogers STEM Scholarship
+            <ul className="stem-ad-list-secondary">
+              <li className="stem-ad-list-item">
+                According to your service and school data records on file, as of{' '}
+                {date}, you have less than 6 months of Post-9/11 GI Bill
+                (Chapter 33) benefits remaining.
+              </li>
+              <li className="stem-ad-list-item">
+                By law, all Rogers STEM Scholarship recipients must have 6
+                months or less of Post-9/11 GI Bill (Chapter 33) benefits left
+                (38 U.S. Code § 3320).
+              </li>
+            </ul>
+          </li>
+        )}
```

Comment:
This is not needed because it is not possible when the only reason for denial is > 180.